### ISSUE TITLE
in Test tab next button is always enabled

### ISSUE THREAD

**Issue by hossamfawzy96** [SEGUC17/Foobar#28]
1: Severity: (low)

2:Reported: by Hossam

3:Description: on taking test and no of the options is checked, the next button is enabled

4: Steps to reproduce the issue: Test then press next without checking any of the options

5: Expected result: display error "no option checked " or next button is disabled

**Comment by markarsanious** [comment]
yeah it is not a must to answer all the questions. It is just an extra feature to help you with some suggestions. If you don't want to answer a certain question, you can definitely proceed with the next one. It is not an obligatory questions after all, intended to just help you with what you want to answer.
